app/views/courses/CourseVideosComponent.vue

```diff
@@ -53,6 +54,14 @@ export default Vue.extend({
       type: String,
       default: null
     }
+  },
+  data: () => ({
+	  videoUrls: []
```

Comment:
Can we make this an object with key value pairs?
Pushing onto an array makes everything dependent on the index. This means we lose the information of the order.